src/Config.js

```diff
@@ -220,6 +224,14 @@ export class Config {
     }
   }
 
+  static validateMaxLimit(maxLimit) {
+    if (maxLimit) {
```

Comment:
Actually I reverted the indentation updates as they were triggered by a new version of eslint.  The CI servers don't have that version yet so builds of my last set of changes were failing.